components/credentials/credentials-form.tsx

```diff
@@ -139,10 +133,14 @@ export const CredentialsForm = () => {
 			setIsSubmitting(true);
 			setError(null);
 			await createSchemaAndThenDefinition();
-			// Handle success (e.g., reset form, show success message)
+
+			toast.success("Credential created successfully!");
+
+			form.reset();
+			window.location.reload();
```

Comment:
this needs to be removed,
Please take a look on react-hook-form to create your form
for any API request use react-query, you can revalidate query cache without refreshing the screen